torch/csrc/jit/register_c10_ops.cpp

```diff
@@ -16,11 +23,13 @@ Operator createOperatorFromC10(const c10::OperatorHandle& op) {
       for (auto iter = stack.end() - input_size; iter != stack.end(); ++iter) {
         // TODO Remove the .defined() check once we don't have undefined tensors on the stack anymore (@wanchaol is working on this)
         if (iter->isTensor() && iter->toTensor().defined()) {
```

Comment:
If your invariant is that Tensors must all come first, shouldn't you break as soon as you reach the first non-tensor argument?

Reply:
That's the invariant for wrapped caffe2 operators. Native c10 operators don't have this restriction.